routers/routes/routes.go

```diff
@@ -267,6 +267,7 @@ func RegisterRoutes(m *macaron.Macaron) {
 		m.Get("", func(ctx *context.Context) {
 			ctx.Redirect(setting.AppSubURL + "/explore/repos")
 		})
+		m.Get("/avatars/:username/:size", routers.ExploreAvatars)
```

Comment:
imho it should be better placed under `/user` group:
https://github.com/go-gitea/gitea/blob/8246fe03968a6939159b65371570749f1698439c/routers/routes/routes.go#L403
```

so URL would look like `/user/avatar/:username/:size`

Reply:
You have added to wrong "/user" group. See my previous link above for correct place

Reply:
Now moved